assets/xml/objects/object_mizu_objects.xml

```diff
@@ -0,0 +1,39 @@
+<Root>
+    <File Name="object_mizu_objects" Segment="6">
+        <DList Name="gObjectMizuObjectsMovebgDL_000190" Offset="0x0190"/>
+        <Collision Name="gObjectMizuObjectsMovebgCol_0003F0" Offset="0x03F0"/>
```

Comment:
Could you try to add a name to dlists and collisions?
You can ask in the discord server if you need help.